api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -143,7 +143,7 @@ public Workspace apply(org.pmiops.workbench.db.model.Workspace workspace) {
 
   private static final ResearchPurpose createResearchPurpose(org.pmiops.workbench.db.model.Workspace workspace) {
     ResearchPurpose researchPurpose = new ResearchPurpose()
-        .diseaseFocusedResearch(workspace.getDiseaseFocusedResearch())
+        .diseaseFocusedResearch(workspace.getDiseaseOfFocus() != null && !workspace.getDiseaseOfFocus().equals(""))
```

Comment:
In java, I prefer `isEmpty()` to raw string comparisons.

Reply:
Will do.